Collebra/src/locators/Hello.java

```diff
@@ -0,0 +1,9 @@
+package locators;
+
+public class Hello {
+
+	public static void main(String[] args) {
+		System.out.println("hello le navvya");
+	}
+
```

Comment:
where is the script

Reply:
ok sir i will edit as soon as possible

Reply:
its ok do if fast

Reply:
ok sir thank you

Reply:
other wise u won't get salary of this month